.github/workflows/pal-consensus-review.yml

```
@@ -0,0 +1,313 @@
+          }
+          EOF
+
+      - name: PAL Consensus Review (AWS Bedrock - Primary)
+        if: env.BEDROCK_CONFIGURED == 'true'
+        id: bedrock_review
+        uses: anthropics/claude-code-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          use_bedrock: "true"
+          show_full_output: true
+          max_turns: 25
```

actionlint (1.7.10)

input "max_turns" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_prefix", "claude_args", "claude_code_oauth_token", "github_token", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_foundry", "use_sticky_comment", "use_vertex"

(action)